rasa_core/nlg/template.py

```diff
@@ -76,7 +77,10 @@ def _fill_template_text(
         # Filling the template variables in the template
         if template_vars:
             try:
-                template["text"] = template["text"].format(**template_vars)
+                # blacklist characters that probably should not be
+                # part of slot name and replace with old %-formatting
+                text = re.sub(r'{([^\n,]+?)}', r'{0[\1]}', template["text"])
```

Comment:
@naoko I thought we agreed on not blacklisting the characters but instead handling it as it is done here https://stackoverflow.com/questions/7934620/python-dots-in-the-name-of-variable-in-a-format-string#comment9695339_7934969 ?

Reply:
My attempt was to yes handling showing in stackoverflow. this regex sub is to change `{tag}` to `{0[tag]}`. the only blacklisting I'm doing here is to make sure not to pick up "tag name" with newline and "," as I guess this could be bad assumption but I thought tag name (slot name with new line or , ) would make no sense.